watermark_image:
  output_path: "/opt/ros_ws/src/ros_audit_image/images/watermarked/"
  base_image_path: "/opt/ros_ws/src/ros_audit_image/images/IMG_3431.jpeg"
  font_file_path: "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf"
  font_size: 1
  text_position:
    x: 100
    y: 670
  newline_size: 40
  text_bold_size: 2
  text_color:
    red: 255
    green: 255
    blue: 255
